front/src/common/IntervalsEditor/IntervalsEditor.tsx

```diff
@@ -196,11 +197,16 @@ export const IntervalsEditor: React.FC<IntervalsEditorProps> = (props) => {
             onResize={(index, gap, finalized) => {
               setMode(!finalized ? 'resizing' : null);
               try {
-                const { result, newIndexMapping } = resizeSegment(data, index, gap, 'end');
+                const { result, newIndexMapping } = resizeSegment(data, index, gap, 'end', false);
+                const fixedResults = fixLinearMetadataItems(result?.filter(notEmpty), totalLength, {
+                  fieldName: 'value',
+                  defaultValue,
+                });
+
```

Comment:
Not sure if it is wanted but when you resize the end of the "existing segment" left to the last segment (the empty one) of the editor, this last one is transformed into an "existing segment". Seems like a strange behaviour.
It means that if we want to only resize the before last segment, it transforms the last one and we have to delete it afterwards.

Reply:
Ok nevermind, it seems that it's fixed in the next commit :)